astroid/protocols.py

```diff
@@ -274,7 +274,14 @@ def _resolve_looppart(parts, assign_path, context):
 
 
 @decorators.raise_if_nothing_inferred
-def for_assigned_stmts(self, node=None, context=None, assign_path=None):
+def for_assigned_stmts(
+    self: Union[nodes.For, nodes.Comprehension],
+    node: Union[
+        nodes.List, nodes.Tuple, nodes.AssignName, nodes.AssignAttr, None
+    ] = None,
+    context: Optional[InferenceContext] = None,
+    assign_path: Optional[List[int]] = None,
+):
```

Comment:
It does make sense not to include return types for now. What about adding `Any` until then?